core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-create/container-history/container-history.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnChanges, OnInit, ViewChild } from '@angular/core';
+import { IframeComponent } from '@components/_common/iframe/iframe-component';
+
+@Component({
+    selector: 'dot-container-history',
+    templateUrl: './container-history.component.html',
+    styleUrls: ['./container-history.component.scss']
+})
+export class ContainerHistoryComponent implements OnInit, OnChanges {
```

Comment:
you're missing the "Dot-" prefix also
https://github.com/dotCMS/core/pull/23018#discussion_r980376434

please check that on all Container files

Reply:
@zulqarnainvd